airflow/models/taskinstance.py

```diff
@@ -1362,6 +1362,7 @@ def _run_raw_task(
             return
         except AirflowSkipException as e:
             # Recording SKIP
+            # This change is in reference to [AIRFLOW-5653][CX-16266]
```

Comment:
This CP came from open source which is already available now. Please remove this line.